eo-runtime/src/test/java/org/eolang/AtCompositeTest.java

```diff
@@ -37,14 +38,17 @@
 public final class AtCompositeTest {
 
     /**
-     * Empty message for JUnit Assertions.
-     *
-     * @todo #2297:60m Replace all appearances of {@link AtCompositeTest#TO_ADD_MESSAGE} field in
-     *  eo-runtime with meaningful assert messages. Don't forget to remove
-     *  {@link AtCompositeTest#TO_ADD_MESSAGE} field and remove public modifier from this class if
-     *  no longer need.
+     * Supplier for failed assert messages.
+     * <p>
+     * {@link AtCompositeTest#FAILED_ASSERT_MESSAGE_SUPPLIER} field in
+     * eo-runtime for getting failed assert messages with class and test
      */
-    public static final String TO_ADD_MESSAGE = "TO ADD ASSERTION MESSAGE";
+    public static final Supplier<String> FAILED_ASSERT_MESSAGE_SUPPLIER =
```

Comment:
@ZacParize I think we don't need this supplier now because it actually does not bring anything useful to our tests. Also here you removed `todo` from the source code, which means the issue will be removed from GitHub. So I would revert it back. And as a result you'll get a smaller pull request with a few assert messages changed

Reply:
I got it. I am going to remove supplier at all.
But the goal is changing all messages "TO ADD ASSERTION MESSAGE" in all places.
Please, don't revert, i will fix it.

Reply:
@ZacParize I meant "if I were you I would revert/fix it back". Yes, the goal is to change messages "TO ADD ASSERTION MESSAGE". But with your current changes we'll need the issue "Replace all `AtCompositeTest.FAILED_ASSERT_MESSAGE_SUPPLIER.get(),` in all places") There's no a big difference